_docs-2/administration/upgrading.md

```diff
@@ -38,6 +56,60 @@ will then need to transfer the contents of the current individual files to this
 **Warning**: Upgrading a previously encrypted instance is not supported as the implementation
 and properties have changed.
 
+## Upgrading from 2.0 to 2.1
```

Comment:
Should the title here be "Upgrading from 1.10 or 2.0 to 2.1"? Upgrading from 1.10 is supported and tested IIRC

Reply:
Is it our official policy to support upgrades from 1.10 to 2.1?  I was not sure and modeled the text to our past statements that we do not support multiple leaps with one upgrade.  I'm fine if that's true and my internal testing usually goes from 1.10 to 2.1 - but that's using uno so I was uncertain if there configuration / property file settings in a cluster that could cause issues.  Functionally, 1.10 to 2.1 is fine - its the supporting scripts / config files that I am fuzzy on.

Reply:
IIRC, in discussions with Christopher, upgrades from 1.10 to 2.1 should work.

Reply:
I do not dispute they work.  But do we want to officially declare that?  I'll change the text to reflect that - we can modify it if that is not our official position.

Reply:
Yes, we do want to officially declare that. The two supported upgrade paths are: `LTM.latest --> (LTM+1).latest` and `major.minor.latest --> major.(minor+1).latest`. So, for here, we support 2.0.1 to 2.1.0, and 1.10.2 to 2.1.0. However, the upgrade instructions might be slightly different for 1.10 than for 2.0.